Cargo.toml

```diff
@@ -20,7 +20,7 @@ ordinal = "*"
 serde = { version = "1.0", features = ["derive"] }
 serde_json = "1.0"
 csv = "1.1"
-rocket = "0.4.10"
```

Comment:
`"0.4.10"` resolves to `0.4.10 <= x < 0.5`, whereas `"~0.4"` resolves to `0.4 <= x < 0.5`. Prefer the former

Reply:
of course you should really set it to "0.4.11" because of the bug you linked